report/src/main/java/com/hartwig/actin/report/pdf/tables/molecular/MolecularCharacteristicsGenerator.java

```diff
@@ -142,11 +172,11 @@ private Cell createTMBStatusCell(@Nullable Boolean hasHighTumorMutationalBurden,
     }
 
     @NotNull
-    private Cell createMSStabilityCell(@Nullable Boolean isMicrosatelliteUnstable) {
```

Comment:
Why package-private rather than private?

Reply:
The package-private cells are used in the molecular summary as well.

Reply:
Clear, missed that